exercises/practice/leap/.approaches/cond/snippet.txt

```diff
@@ -0,0 +1,8 @@
+def leap_year?(year) do
+    cond do
+        divides?(year, 400) -> true
+        divides?(year, 100) -> false
+        divides?(year, 4) -> true
+        true -> false
```

Comment:
The function `divides?` is defined in the introduction document and is shared between approaches. Because of that, I don't think repeating it in individual snippets will be of value. The snippets, as I understand, should illustrate the approach, and they don't have to compile as such. It's just a code fragment. The reason I'm using the `divedes?` function is so that `rem(year, 100) == 0)` doesn't distract from what really differs.